frontend/src/app/modules/landing/registration/registration.component.sass

```diff
@@ -0,0 +1,110 @@
+.registration-container
+    height: 100%
+    padding: 0 20%
+    display: flex
+    flex-direction: column
+    align-items: center
+    justify-content: space-between
+
+.registration-content
+    width: 100%
+    margin-top: 100px
+
+    h4
+        color: #282828
```

Comment:
Move it to color.sass file, please
Yeah, you need to create file with global variables

Reply:
Done, let ne know if there are some problem